egs/librimix/ConvTasNet/run.sh

```diff
@@ -95,10 +110,30 @@ if [[ $stage -le 1 ]]; then
 	echo "librimix/ConvTasNet" > $expdir/publish_dir/recipe_name.txt
 fi
 
+
 if [[ $stage -le 2 ]]; then
 	echo "Stage 2 : Evaluation"
-  $python_path eval.py --exp_dir $expdir --test_dir $test_dir \
+
+	if [[ $compute_wer -eq 1 ]]; then
+	  if [[ $eval_mode != "max" ]]; then
+	    echo "Cannot compute WER without max mode. Start again with --stage 2 --compute_wer 1 --eval_mode max"
+	    exit 1
+	  fi
+    # Install espnet if not instaled
+    if not python -c "import espnet" &> /dev/null; then
+        echo 'This recipe requires espnet. Installing requirements.'
+        $python_path -m pip install espnet_model_zoo
+        $python_path -m pip install jiwer
+    fi
+  fi
```

Comment:
I never go through this loop even when i don't have espnet installed . Not sure why.

Reply:
have you tried `if ! python -c "import espnet" &> /dev/null`? I don't think `not` is a valid bash keyword